include/swift/SIL/SILValue.h

```diff
@@ -285,6 +285,12 @@ class ValueBase : public SILNode, public SILAllocated<ValueBase> {
   /// and it has a single consuming user. Returns .none otherwise.
   inline Operand *getSingleConsumingUse() const;
 
+  inline SmallVector<Operand *, 4>
```

Comment:
Can you use an iterator here instead of returning a SmallVector?

Also why do you need the verifying argument?

Reply:
When verifying with OwnershipVerifierTextualErrorDumper - we don't want this assert to fire: https://github.com/apple/swift/blob/58d07a4efd0d9e5a38dc4b0d029d6f80d7152a62/include/swift/SIL/SILValue.h#L586. That reminds me that callers should pass `true` only when verifying with OwnershipVerifierTextualErrorDumper.